src/app/shared/components/template/services/instance/template-action.service.ts

```diff
@@ -228,6 +228,8 @@ export class TemplateActionService extends SyncServiceBase {
         return processor.processTemplateWithoutRender(templateToProcess);
       case "google_auth":
         return await this.authService.signInWithGoogle();
+      case "auth_sign_out":
+        return await this.authService.signOut();
```

Comment:
nit(non-blocking) appreciate this is likely just to use with some testing templates, but I think might be good to create a follow-up issue to tidy this up a bit. I'd say ideally `google_auth` action should just be replaced by an `auth` action namespace that includes both `sign_in` and `sign_out` child actions.